<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="txt_Nombre">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Dirección de Internet</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Dirección fìsica</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Telèfono</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="txt_Dinternet">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="txt_Dfisica">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="txt_telefono">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>INFORMACION DE LA TIENDA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="boton_Cargar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar desde archivo bInario</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_Guardar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar en archivo bInario</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>201</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="boton_Agregar">
       <property name="text">
        <string>Agregar producto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="boton_Modificar">
       <property name="text">
        <string>Modificar Producto</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="boton_Eliminar">
       <property name="text">
        <string>Eliminar producto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="boton_CrearTienda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear tienda</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listaProductos">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>441</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
